code/modules/tgui/external.dm

```diff
@@ -158,7 +158,7 @@
 	// Name the verb, and hide it from the user panel.
 	set name = "uiclose"
 	set hidden = TRUE
-	var/mob/user = src && src.mob
+	var/mob/user = src?.mob
```

Comment:
Out of scope of this PR, but this check is very weird. Then again, clients, they are arcane aplenty.

Reply:
Do you want me to revert the change? This is indeed a weird check.

Reply:
Eh, keep it or revert it, up to you. Feel free to resolve this conversation.